Documentation/CRDs/Shared-Filesystem/ceph-fs-subvolumegroup-crd.md

```diff
@@ -64,3 +64,21 @@ If any setting is unspecified, a suitable default will be used automatically.
 !!! note
     Only one out of (export, distributed, random) can be set at a time.
     By default pinning is set with value: `distributed=1`.
+
+## How to set a StorageClass to consume CephFilesystemSubVolumeGroup
+
+* Create a CephFilesystem CR 
+* Create a CephFilesystemSubvolumegroup CR 
+* Extract the `clusterID` from the status of the CephFilesystemSubvolumegroup CR:
+
+    ```bash
+    kubectl -n rook-ceph get cephfilesystemsubvolumegroup -o yaml
+    ```
+    
+    ```yaml
+    status:
+    info:
+    clusterID: 80fc4f4bacc064be641633e6ed25ba7e
+    ```
+
+* Set the `clusterID` in the storage class to this value instead of the name of the cluster namespace
```

Comment:
Can you also specify to not only set this in storageclass but also volumesnapshotclass, volumegroupsnapshotclass as well.